pyproject.toml

```diff
@@ -56,5 +56,5 @@ dev = [
 addopts = "--resetdb"
 
 [project.scripts]
-install-labels = "scripts.neomodel_install_labels:main"
-remove-labels = "scripts.neomodel_remove_labels:main"
+neomodel_install_labels = "scripts.neomodel_install_labels:main"
```

Comment:
Does it still work with the latest renaming of the script files to "neomodel_install_labels.py" ? (See latest commit in master branch)

Reply:
@mariusconjeaud the scripts have always been `neomodel_install_labels` and `neomodel_remove_labels` and any user scripts still using those names for management would fail.

This change just brings back the original names (and is tested and works)

Reply:
OK great ! Then we're all good.